[haskell/en] Minor changes to Haskell document that were bothering me (#5207)
adambard/learnxinyminutes-docs: haskell.md

['H', 'e', 'l', 'l', 'o'] -- "Hello"

-- Lists can be indexed with the `!!` operator followed by an index

> -- Be aware that since lists are linked lists, this is an O(n) operation

Reviewer: ```suggestion
-- but this is an O(n) operation because lists are linked lists
```
